Add CITE post-processing to docs (#110)
AlexsLemonade/scpca-docs: docs/processing_information.md

> <!-- START doctoc generated TOC please keep comment here to allow auto update -->
> <!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
> **Table of Contents**
> 
> - [Processing information](#processing-information)
>   - [Single-cell and single-nuclei RNA-seq](#single-cell-and-single-nuclei-rna-seq)
>     - [Mapping and quantification using alevin-fry](#mapping-and-quantification-using-alevin-fry)
>       - [Reference transcriptome index](#reference-transcriptome-index)
>       - [Selective alignment](#selective-alignment)
>       - [Alevin-fry parameters](#alevin-fry-parameters)
>     - [Post alevin-fry processing](#post-alevin-fry-processing)
>       - [Combining counts from spliced cDNA and intronic regions](#combining-counts-from-spliced-cdna-and-intronic-regions)
>       - [Filtering cells](#filtering-cells)
>     - [Processed gene expression data](#processed-gene-expression-data)
>   - [CITE-seq quantification](#cite-seq-quantification)
>     - [Combining ADT counts with RNA counts](#combining-adt-counts-with-rna-counts)
>     - [Processed CITE-seq data](#processed-cite-seq-data)
>   - [Multiplexed libraries](#multiplexed-libraries)
>     - [Hashtag oligonucleotide (HTO) quantification](#hashtag-oligonucleotide-hto-quantification)
>     - [HTO demultiplexing](#hto-demultiplexing)
>     - [Genetic demultiplexing](#genetic-demultiplexing)
>   - [Spatial transcriptomics](#spatial-transcriptomics)
>     - [Mapping and quantification using Space Ranger](#mapping-and-quantification-using-space-ranger)
>   - [Bulk RNA samples](#bulk-rna-samples)
>     - [Preprocessing with fastp](#preprocessing-with-fastp)
>     - [Mapping and quantification using salmon](#mapping-and-quantification-using-salmon)
>       - [Salmon parameters](#salmon-parameters)
> 
> <!-- END doctoc generated TOC please keep comment here to allow auto update -->
> 

Reviewer: TOC and indexing is handled by ReadTheDocs, so no doctoc
```suggestion
```

Cells with a high likelihood of being compromised (greater than 0.75) and cells that do not pass a minimum number of unique genes detected threshold of 200 are removed from the counts matrix present in the `_processed.rds` file.

If CITE-seq data is present, additional low-quality cells are further removed based on antibody-derived tag (ADT) content, as described in [Processed CITE-seq data](#processed-cite-seq-data).

Log-normalized counts are calculated using the deconvolution method presented in [Lun, Bach, and Marioni (2016)](https://doi.org/10.1186/s13059-016-0947-7).
The log-normalized counts are used to model variance of each gene prior to selecting the top 2000 highly variable genes (HVGs).
These HVGs are then used as input to principal component analysis, and the top 50 principal components are selected.
Finally, these principal components are used to calculate the [UMAP (Uniform Manifold Approximation and Projection)](http://bioconductor.org/books/3.13/OSCA.basic/dimensionality-reduction.html#uniform-manifold-approximation-and-projection) embeddings.

> ### Processed CITE-seq data

Reviewer: I would move this section to a subsection of `CITE-seq quantification`, as it seems odd to talk about the filtering before quantification.

Author: Done in https://github.com/AlexsLemonade/scpca-docs/pull/110/commits/1e3f1c4b35474acbca477d047671462dcc52d848 (w/ cleanup in https://github.com/AlexsLemonade/scpca-docs/pull/110/commits/c77beae7761604bc700c1e47a5ce241374595822 ).

If the experiment contains CITE-seq data, further filtering steps are applied to the `SingleCellExperiment` object found in `_processed.rds`.
An ambient profile representing antibody-derived tag (ADT) proportions present in the ambient solution is calculated from the `_unfiltered.rds` object using [`DropletUtils::ambientProfileEmpty()`](https://rdrr.io/github/MarioniLab/DropletUtils/man/ambientProfileEmpty.html).

> This ambient profile is then as input to calculate quality-control statistics using [`DropletUtils::cleanTagCounts()`](https://rdrr.io/github/MarioniLab/DropletUtils/man/cleanTagCounts.html).
> If negative control ADTs are present, this information is also provided to `DropletUtils::cleanTagCounts()`.

Reviewer: For some reason I want this as one sentence.
```suggestion
This ambient profile, along with negative control information, if present, is then as input to calculate quality-control statistics using [`DropletUtils::cleanTagCounts()`](https://rdrr.io/github/MarioniLab/DropletUtils/man/cleanTagCounts.html).
```

> `DropletUtils::cleanTagCounts()` identifies cells that should be discarded as those with overly high levels of ambient contamination and/or total negative control counts.
> When cells were filtered based on ADT content, the RNA count matrix was also filtered to match such that the same cells are present in both experiments.

Reviewer: Shortening this up a bit. I hope that the fact that we are removing data for _cells_ implies that we are removing all data. But I could be convinced that we do need a second sentence clarifying this, but I would avoid the "both experiments" phrasing, as I think that could be confusing as it refers to the data structure, not real "experiments"

```suggestion
Cells identified by `DropletUtils::cleanTagCounts()` as having high levels of ambient contamination and/or negative control tags are are removed from the counts matrices present in the `_processed.rds` file.
```

Author: I accepted this and am changing slightly from "from the counts matrices" --> "from all the counts matrices" which I think helps get the point across
